aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1486,6 +1492,26 @@ impl AptosVM {
         Ok(())
     }
 
+    /// Check whether the bytecode can be published to mainnet based on the unstable tag in the metadata
+    fn filter_unstable_bytecode(&self, modules: &[CompiledModule]) -> VMResult<()> {
```

Comment:
Personally I find the name "filter" a bit confusing -- Rust usually uses this word to denote the action of removing some items from a collection but keeping the rest, but here we're just performing a check.

Reply:
how about `reject_unstable_bytecode` instead?